embedded/sqlutils/params.go

```diff
@@ -0,0 +1,66 @@
+package sqlutils
```

Comment:
I'd move under api/schema package, otherwise it would introduce a dependency to the embedded one